edx_sga/static/css/edx_sga.css

```diff
@@ -1,13 +1,16 @@
 /* CSS for StaffGradedAssignmentXBlock */
 
+.modal-iframe-height {
+    max-height: 600px;
+}
+
```

Comment:
I guess that is changing the legacy behavior, but not sure. You need to confirm it